Celeste.Mod.mm/Patches/MapEditor.cs

```diff
@@ -85,7 +85,7 @@ public patch_MapEditor(AreaKey area, bool reloadMapData = true)
             }
 
             session.Level = level.Name;
-            session.RespawnPoint = customRespawnPoint ? at : null;
+            session.RespawnPoint = customRespawnPoint ? at : new Vector2?();
```

Comment:
I wish I knew why this was necessary, but dotnet refused to compile everest until I made this change.

Reply:
You might be able to make it `default`.